docs/setup.md

```diff
@@ -0,0 +1,28 @@
+| [Home](../README.md) |
+|----------------------|
+
+# Installation
+
+1. To install a widget, click **Content Hub** > **Discover**.
+
+2. From the list of widgets that appears, search **Playbook Buttons**.
+
+3. Click the **Playbook Buttons** widget card.
+
+4. Click **Install** on the lower part of the screen to begin the installation.
+
+# Configuration
+
+The **Playbook Buttons** widget allows you to customize its behavior to suit your specific requirements. Here are the available configuration settings:
+
+| Fields                                     | Description                                                                                              |
+|--------------------------------------------|----------------------------------------------------------------------------------------------------------|
+| Select Action Buttons                      | Select playbooks to execute from the record view panel. Select icons for each.                           |
+| Show Playbook Execution Progress On Wizard | Select this option to view selected playbooks' execution on the playbook execution wizard.               |
```

Comment:
Over here also add a note, that to use this option user should install PB execution wiz. Otherwise this feature will not work.

Reply:
added